lib/pbkdf2.dart

```diff
@@ -31,33 +30,67 @@ class PBKDF2 {
     return key;
   }
 
-  List<int> _computeBlock(String password, String salt, int iterations, int blockNumber) {
-    var hmac = new HMAC(hash, password.codeUnits);
-    hmac.add(salt.codeUnits);
-    _writeBlockNumber(hmac, blockNumber);
-    var lastDigest = hmac.close();
-    var result = lastDigest;
+  int get prfLengthInBytes {
```

Comment:
This looks a little inefficient, calling this every time you want to get this value.